pkg/recipes/configloader/environment.go

```diff
@@ -114,6 +114,17 @@ func getConfiguration(environment *v20231001preview.EnvironmentResource, applica
 
 	if environment.Properties.RecipeConfig != nil {
 		config.RecipeConfig = envDatamodel.Properties.RecipeConfig
+		secrets := config.RecipeConfig.Terraform.Authentication.Git.PAT
```

Comment:
This is specific to Terraform, right? Where do we check if this recipe is a Terraform recipe?